conf/secant.conf

```diff
@@ -32,4 +32,7 @@ IMAGE_LIST_URL=https://example.org/secant/
 STATE_DIR=/var/lib/secant
 
 # The root of secant installation
-SECANT_PATH=/opt/secant
+#SECANT_PATH=/opt/secant
```

Comment:
why to comment out the variable (remember it's just a template that is adapted for each deployment)?